.env.example

```
@@ -50,10 +50,11 @@ GITHUB_PERSONAL_ACCESS_TOKEN=
 GITHUB_HOST=
 GITHUB_OWNER=
 GITHUB_REPO=
-# Optional dashboard account connection. Create a GitHub OAuth App, enable Device Flow,
-# and put its public client ID here. Tokens are stored locally in .agent-team/github-auth.json.
+# Optional dashboard account connection. Prefer fine-grained PAT or GitHub App
+# permissions for Contents, Pull requests, Actions, and Metadata. OAuth scopes
+# stay blank by default to avoid broad org/repo grants.
 GITHUB_OAUTH_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_OAUTH_CLIENT_ID key should go before the GITHUB_OWNER key

(UnorderedKey)